packages/js/test/plugins/auctionHouseModule/createAuctionHouse.test.ts

```diff
@@ -143,3 +150,84 @@ test('[auctionHouseModule] create new Auction House with SPL treasury', async (t
     },
   });
 });
+
+test('[auctionHouseModule] create new Auctioneer Auction House', async (t: Test) => {
+  // Given we have a Metaplex instance.
+  const mx = await metaplex();
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // When we create a new Auctioneer Auction House.
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({
+      sellerFeeBasisPoints: 200,
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+    })
+    .run();
+
+  // Then the new Auction House has Auctioneer attached.
+  t.ok(auctionHouse.hasAuctioneer);
+
+  // And the Auctioneer PDA for that Auction House was created.
+  const ahAuctioneerPda = findAuctioneerPda(
+    auctionHouse.address,
+    auctioneerAuthority.publicKey
+  );
+  const ahAuctioneerAccount = await mx.rpc().getAccount(ahAuctioneerPda);
+  t.ok(ahAuctioneerAccount.exists);
+});
+
+test('[auctionHouseModule] create new Auctioneer Auction House with separate authority', async (t: Test) => {
+  // Given we have a Metaplex instance.
+  const mx = await metaplex();
+
+  const auctioneerAuthority = Keypair.generate();
+  const authority = Keypair.generate();
+
+  await mx.rpc().airdrop(authority.publicKey, sol(100));
+
+  // When we create a new Auctioneer Auction House with a separate authority.
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({
+      sellerFeeBasisPoints: 200,
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+      authority,
+    })
+    .run();
+
+  // Then the new Auction House has separate authority.
+  t.equal(
+    auctionHouse.authorityAddress.toBase58(),
+    authority.publicKey.toBase58()
+  );
+  // And Auctioneer was delegated.
+  t.ok(auctionHouse.hasAuctioneer);
```

Comment:
No broken windows. 🙈 😅

```suggestion
  );

  // And Auctioneer was delegated.
  t.ok(auctionHouse.hasAuctioneer);
```